### PR TITLE
Remove non-breaking spaces from corelight dashboards

### PR DESCRIPTION
**Pull request by SaaldjorMike**
Seems like a few non-breaking spaces made it into the file `corelight-dashboard-conn.json`. It seems Humio does not like them. Replacing them by regular spaces seems to load just fine.

Screenshot 2019-07-08 at 16 27 02

I also removed the directory `__MACOSX` inside the zip file.

### CODE REVIEW

**Comment by mwl** [comment]
We shouldn't have zip files containing json files in our repo. Feel free to zip/tarball it during build in the Makefile file

File: Makefile [hunk 1]
```diff
@@ -7,10 +7,6 @@ run: deps
 	# CSS gets mashed if we don't use --disableFastRender
 	hugo server --disableFastRender
 
-run-docker: deps
-	# Runs hugo server in a docker container, container is automatically destroyed when stopped
-	bash run-hugo-docker.sh
```
Comment (SaaldjorMike) [inline Makefile]: According to the git log, that file was removed last year 😄

File: Makefile [hunk 2]
```diff
@@ -21,9 +17,13 @@ data/functions.json:
 	curl -fs https://repo.humio.com/repository/maven-releases/com/humio/docs/queryfunctions/$(RELEASE)/queryfunctions-$(RELEASE).json > data/functions.json
 
 data/metrics.json:
-	curl -fs https://repo.humio.com/repository/maven-releases/com/humio/docs/metrics/$(RELEASE)/metrics-$(RELEASE).json > data/metrics.json	
+	curl -fs https://repo.humio.com/repository/maven-releases/com/humio/docs/metrics/$(RELEASE)/metrics-$(RELEASE).json > data/metrics.json
+
+public/zeek-files:
```
Comment (mwl) [inline Makefile]: 2 things :)
Don't forget to delete the file in the `clean` target.
Second, we use the intended Make convention of naming the target after the output, so `public/zeek-files/corelight-dashboards.zip`

File: corelight-dashboards/corelight-dashboard-conn.json [hunk 3]
```diff
@@ -0,0 +1 @@
...s":120,"showQueryEventDistribution":false,"start":"24h"},"options":{"x":4,"y":6,"width":4,"height":4,"visualisation":{"widgetType":"table-view"}}},{"id":"c5c1ab46-6aff-4bc9-8356-f09908375270","title":"Top Services","query":{"queryString":"#path=conn is_broadcast!=true | groupby(service) | sort()","isLive":true,"timeZoneOffsetMinutes":120,"showQueryEventDistribution":false,"start":"24h"},"options":{"x":0,"y":6,"width":4,"height":4,"visualisation":{"widgetType":"pie-chart"}}},{"id":"c8d38299-325a-4b4c-bb7f-e76a3a58d413","title":"Largest transfers between host/port pairs","query":{"queryString":"#path=conn\n| (proto=tcp OR (proto=udp AND orig_pkts>0 AND resp_pkts>0))\n| conn_state != \"S0\"\n| groupby([id.orig_h, id.resp_h, id.resp_p, proto],\n          function=[sum(resp_bytes, as=resp_bytes_sum),\n                    sum(orig_bytes, as=orig_bytes_sum),\n                    count(as=number_of_conns)])\n| eval(total_bytes=orig_bytes_sum+resp_bytes_sum)\n| sort(total_bytes, limit=20)\n| rdns(id.resp_h)","isLive":true,"timeZoneOffsetMinutes":120,"showQueryEventDistribution":false,"start":"24h"},"options":{"x":0,"y":10,"width":12,"height":5,"visualisation":{"widgetType":"table-view"}}}]}
```
Comment (mwl) [inline corelight-dashboards/corelight-dashboard-conn.json]: Pop it in `artefacts/corelight-dashbards/…`.

File: Makefile (hunk 1 above)
Comment (mwl) [reply Makefile]: Delete the target

File: Makefile (hunk 2 above)
Comment (SaaldjorMike) [reply Makefile]: For the first thing, the `clean` target already removes the entire `public` directory.
For the second thing, I pushed a change to fix that.

**Pull request merged**